spec/system/anybody_register_account_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe '誰でもアカウントを登録できる', type: :system do
+  context '全てを適切に入力した場合' do
+    it 'アカウントが登録される', js: true do
+      visit root_path
+      click_link 'アカウント登録'
+      fill_in 'ユーザー名', with: 'tester'
+      fill_in 'Eメール', with: 'tester@example.com'
+      fill_in 'プロフィール', with: 'I am a tester'
+      fill_in 'ブログURL', with: 'https://github.com/tester'
+      fill_in 'パスワード *', with: 'password'
+      fill_in 'パスワード（確認用） *', with: 'password'
+
+      expect { click_button 'アカウント登録' }.to change { User.all.count }.from(0).to(1)
```

Comment:
```suggestion
      expect { click_button 'アカウント登録' }.to change(User, :count).by(1)
```

こういう書き方もできます。（知ってるかもだけど）

Reply:
そっちの方がシンプルですね。